lib/util/ast.js

```diff
@@ -28,7 +28,7 @@ function isClassAttribute(node) {
     default:
       name = node.name.name;
   }
-  return /^class(Name)?$/.test(name);
+  return /^class(Name)?$|^tw$/.test(name);
```

Comment:
I'm not familiar with Twin, is the `tw` attribute standard ? 
If not, it would be better to inject the chosen attribute name via an additional setting.

WDYT?

Reply:
@finnmerlett, WDYT about the additional setting ?

Reply:
Yes with Twin `tw` is the prop they've chosen to be their inject point. I think an additional setting would certainly be a better option, but I didn't have the time to craft a PR for that - sorry 😬 . I thought it better to submit what I had rather than nothing at all 🤷

Reply:
For a simpler feature, I've not seen any other custom inject props, so perhaps a simple settings switch to enable it to allow backwards compatibility would be a suitable middle ground

Reply:
I've created a new PR based on the work of @finnmerlett that adds support for this extra options. That makes the implementation more generic. See: https://github.com/francoismassart/eslint-plugin-tailwindcss/pull/179